src/language/kinds.ts

```diff
@@ -1,72 +1,97 @@
+import type {
+  ARGUMENT,
+  BOOLEAN,
+  DIRECTIVE,
+  DIRECTIVE_DEFINITION,
+  DOCUMENT,
+  ENUM,
+  ENUM_TYPE_DEFINITION,
+  ENUM_TYPE_EXTENSION,
+  ENUM_VALUE_DEFINITION,
+  FIELD,
+  FIELD_DEFINITION,
+  FLOAT,
+  FRAGMENT_ARGUMENT,
+  FRAGMENT_DEFINITION,
+  FRAGMENT_SPREAD,
+  INLINE_FRAGMENT,
+  INPUT_OBJECT_TYPE_DEFINITION,
+  INPUT_OBJECT_TYPE_EXTENSION,
+  INPUT_VALUE_DEFINITION,
+  INT,
+  INTERFACE_TYPE_DEFINITION,
+  INTERFACE_TYPE_EXTENSION,
+  LIST,
+  LIST_TYPE,
+  NAME,
+  NAMED_TYPE,
+  NON_NULL_TYPE,
+  NULL,
+  OBJECT,
+  OBJECT_FIELD,
+  OBJECT_TYPE_DEFINITION,
+  OBJECT_TYPE_EXTENSION,
+  OPERATION_DEFINITION,
+  OPERATION_TYPE_DEFINITION,
+  SCALAR_TYPE_DEFINITION,
+  SCALAR_TYPE_EXTENSION,
+  SCHEMA_DEFINITION,
+  SCHEMA_EXTENSION,
+  SELECTION_SET,
+  STRING,
+  UNION_TYPE_DEFINITION,
+  UNION_TYPE_EXTENSION,
+  VARIABLE,
+  VARIABLE_DEFINITION,
+} from './_kinds.js';
+
+export type Kind =
```

Comment:
```suggestion
/**
 * The set of allowed kind values for AST nodes.
 */
export type Kind = 
```

This will have the desired effect of having JSDoc on both the type and term level.